src/qt/settings.cpp

```diff
@@ -0,0 +1,35 @@
+#include <QSettings>
+#include <QString>
+
+#include "settings.hpp"
+
+namespace Settings
+{
+    QString bios_path;
+
+    // If we had a files for QT helper functions, this could go there, but here works for now.
+    // https://wiki.qt.io/Technical_FAQ#How_can_I_convert_a_QString_to_char.2A_and_vice_versa.3F
+    char *as_char(QString q_str)
```

Comment:
Somewhat nitpicky, but have the asterisk next to the type rather than the function.

```suggestion
    char* as_char(QString q_str)
```